LayoutTests/fast/canvas/canvas-createPattern-video-loading.html

```diff
@@ -63,17 +64,12 @@
         });
     }
 
-    function loadeddata()
+    async function firstframe()
     {
         drawImageToCanvasAndCheckPixels();
 
         video.currentTime = 1;
-        video.play();
-    }
-
-    function playing()
-    {
-        video.pause();
+        await Promise.all([ once(video, "seeked"), waitForVideoFrame(video) ]);
```

Comment:
Are we sure setting video.currentTime will not trigger a rvfc on a past frame.
i.e should we do the following: `await once(video, "seemed"); await waitForVideoFrame(video)`